lib/generators/feature_flag/feature_flag_generator.rb

```diff
@@ -0,0 +1,17 @@
+class FeatureFlagGenerator < Rails::Generators::NamedBase
+  REPLACEMENT_REGEX = /(def run).*?(end.*)\z/m
+
+  def create_feature_flag_config
+    append_file "config/feature_flags.yml", %(  - #{file_name}\n)
+  end
+
+  def create_dus
+    script_name = "add_#{file_name}_feature_flag"
+    generate "data_update", script_name, "--no-spec"
+    file = Dir.glob("lib/data_update_scripts/*#{script_name}.rb").first
```

Comment:
This is not super robust but IMO good enough™. Unless someone manually creates scripts with conflicting names we should be ok.